meson.build

```diff
@@ -150,6 +150,21 @@ if get_option('buildtype').startswith('debug')
             '-D_LIBCPP_DEBUG=1',
         ]
     endif
+
+    # Enable Loguru stack traces if supported
+    stacktrace_headers = ['cxxabi.h', 'dlfcn.h', 'execinfo.h']
+
+    all_stacktrace_headers_found = true
+
+    foreach header : stacktrace_headers
+        if not cxx.has_header(header)
+            all_stacktrace_headers_found = false
+        endif
+    endforeach
+
+    if all_stacktrace_headers_found
+        add_project_arguments('-DLOGURU_STACKTRACES=1', language: 'cpp')
+    endif
```

Comment:
Want to move the other two loguru settings over from src/config.h.in?

(For MSVC builds, maybe loguru's auto checks are good enough, or should defines be added in the solution file?)

Reply:
I wonder if we even need the assert redefine, since I'm getting a stack trace just fine without it, and it just does an undef, so `loguru.hpp` would have to be included after every include of `cassert`. I'm not sure it buys us much. From `loguru.hpp':

```cpp
#if LOGURU_REDEFINE_ASSERT
	#undef assert
	#ifndef NDEBUG
		// Debug:
		#define assert(test) CHECK_WITH_INFO_F(!!(test), #test) // HACK
	#else
		#define assert(test)
	#endif
#endif // LOGURU_REDEFINE_ASSERT
```